benchmark/Streamly/Benchmark/Data/Array/Foreign/Mut.hs

```diff
@@ -63,6 +64,8 @@ o_1_space_serial_marray value array =
         $ MArray.strip (\x -> x < value `div` 2 || x > value `div` 2)
     , benchIO "strip (> 0)" (const array)
         $ MArray.strip (> 0)
+    , benchIO "modifyIndices id" (const array)
+        $ Stream.fold (MArray.modifyIndices id array) . (IS.fromStreamD . MArray.toStreamD)
```

Comment:
Can we use `+1` instead of `id`, to ensure some change is being made to the array.

Is this line going beyond 80 columns?